scss/civicrm/common/_forms.scss

```diff
@@ -109,7 +109,8 @@
   .form-layout,
   .form-layout-compressed,
   .crm-accordion-body,
-  .crm-form-block {
+  .crm-form-block,
+  #task-section {
```

Comment:
what's going on with this list? the elements in this list seems to be very different from each (forms, accordions, sections), what is it that they all have in common?

Reply:
This are the selectors under which inputs can be there in CiviCRM,
Basically we are targeting each selectors which has input fields, and in CiviCRM it's not always that the inputs are wrapped with `form` tag

Reply:
Also, we can not directly style on `form` tag as, we are injecting with Resource URL so, no matter what, page it is, Admin or Public, our css will reflect and, it can have potential break on front parts

Reply:
ok could you then write a mixin that you then document? something like
```scss
@include form-containers { // find a more descriptive name
  // the style
}
```

and then mixin will be something like
```scss
@mixin form-containers {
  .form-layout,
  .form-layout-compressed,
  .crm-accordion-body,
  .crm-form-block,
  #task-section {
    @content;
  }
}